### ISSUE TITLE
Multi timeframe

### ISSUE THREAD

**Issue by dkgrif1286** [freqtrade/freqtrade#4320]
The strategy that I'm trying to create uses the 6h timeframe and 3 white soldiers to find a long trend. Then I want the bot to trade on the 1m timeframe with MFI to enter and either MFI or a certain ROI to exit. Is this possible in freqtrade?

```
import numpy
import pandas
from pandas import DataFrame

import talib.abstract as ta
import freqtrade.vendor.qtpylib.indicators as qtpylib

class MFILong(IStrategy):

    minimal_roi = {
        "120": 0.01,
        "60": 0.02,
        "30": 0.03,
        "15": 0.04,
        "0": 0.05
    }

    stoploss = -0.05

    # Trailing stoploss
    trailing_stop = True
    trailing_only_offset_is_reached = True
    trailing_stop_positive = 0.0
    trailing_stop_positive_offset = 0.003

    # Optimal ticker interval for the strategy.
    timeframe = '6h' 
    timeframe = '1m'

    # Run "populate_indicators()" only for new candle.
    process_only_new_candles = False

    # These values can be overridden in the "ask_strategy" section in the config.
    use_sell_signal = True
    sell_profit_only = False
    ignore_roi_if_buy_signal = False

    # Number of candles the strategy requires before producing valid signals
    startup_candle_count: int = 3 

    # Optional order type mapping.
    order_types = {
        'buy': 'market',
        'sell': 'market',
        'stoploss': 'market',
        'stoploss_on_exchange': True
    }

    # Optional order time in force.
    order_time_in_force = {
        'buy': 'gtc',
        'sell': 'gtc'
    }

    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:

        # Momentum Indicators
        # MFI
        dataframe_m1['mfi'] = ta.MFI(dataframe)

        # ROI
        dataframe_m1['roi'] = ta.ROI(dataframe)

        # Pattern Recognition - Bullish/Bearish candlestick patterns
        #Three White Soldiers: values [0, 100]
	dataframe_6h['CDL3WHITESOLDIERS'] = ta.CDL3WHITESOLDIERS(dataframe) # values [0, 100]

        # # Chart type
        # # Heikin Ashi Strategy
        heikinashi = qtpylib.heikinashi(dataframe
        dataframe['ha_open'] = heikinashi['open']
        dataframe['ha_close'] = heikinashi['close']
        dataframe['ha_high'] = heikinashi['high']
        dataframe['ha_low'] = heikinashi['low']

        return dataframe

    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:

	dataframe_6h = resample_to_interval(dataframe, 360)
	dataframe_6h['CDL3WHITESOLDIERS']=ta.CDL3WHITESOLDIERS(dataframe_6h, timeperiod=150)
	dataframe_1m = resample_to_interval(dataframe)
	dataframe_1m['mfi']=ta.mfi(dataframe)

         dataframe.loc[
            (
                (qtpylib.crossed_above(dataframe_1m['mfi'], 18) &
	        (dataframe_6h['CDL3WHITESOLDIERS'] = true)
	    ),
            'buy'] = 1

        return dataframe

    def populate_sell_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:

        dataframe.loc[
            (
                (qtpylib.crossed_above(dataframe_m1['mfi'], 82) | 
	        (dataframe_m1['roi'] = .03)
            ),
            'sell'] = 1 

        return dataframe
```

**Comment by xmatthias** [comment]
you'll not be able to resample 1m to 6h candles and still have 150 6h candles left. 

it may work in backtesting, but not in live mode.

Also, from a quick glance, your stoploss seems too tight, anything below 1% will fail for sure - above that it'll depend on the selected pairs if it works, or needs a wider stop.

**Comment by dkgrif1286** [comment]
Thanks for the response. So is there a way to use multiple timeframes?

**Comment by xmatthias** [comment]
yes for sure as [documented](https://www.freqtrade.io/en/latest/strategy-customization/#additional-data-informative_pairs).

if your distance is too big however, you'll not be able to combine them anymore (not enough datapoints on the fast timeframe).

**Comment by dkgrif1286** [comment]
Hey @xmatthias. Thanks, I must have missed that section. So I'm just looking at the last 3 candles on the 6h chart to find a long trend hence CDL3WHITESOLDIERS and then trade on the 5m chart (update from 1m). I took the code from the doc you provided and I'm getting the attached error. 

![Error](https://user-images.githubusercontent.com/78570577/107157604-04489600-6953-11eb-8296-2128b11a2b63.JPG)
[mfilong3.txt](https://github.com/freqtrade/freqtrade/files/5939837/mfilong3.txt)

Basically, I need the CDLWHITESOLDIERS to be true on the 6h chart and MFI to cross above 18 to enter a trade. Any help would be greatly appreciated. I can't figure this out to save my life.

**Comment by xmatthias** [comment]
The log states it clearly - syntax error on line 100.

Best use an editor with syntax highlighting (like vscode with the python plugin) which will highlight the problem for you (maybe even propose a fix).
